beets/ui/__init__.py

```diff
@@ -74,6 +74,14 @@ class UserError(Exception):
 
 # Encoding utilities.
 
+def _convert_args(args):
+    if not args:
+        return None
+    for i, elem in enumerate(args):
+        if isinstance(elem, bytes):
+            args[i] = elem.decode(_arg_encoding())
+    return args
+
```

Comment:
After reexamining I'm pretty sure this is really only needed for the ui tests that directly call `ui.raw_main` vs use `run_command`